tests/test_data_entry_flow.py

```diff
@@ -19,42 +19,42 @@
 )
 
 
-@pytest.fixture
-def manager():
-    """Return a flow manager."""
-    handlers = Registry()
-    entries = []
+class MockFlowManager(data_entry_flow.FlowManager):
```

Comment:
I moved this out of the `manager` fixture definition, to improve type hints and avoid `attribute-defined-outside-init` warnings.